packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsItem.tsx

```diff
@@ -10,6 +10,7 @@ import withErrorBoundary from '../common/withErrorBoundary'
 import PlusOneIcon from '@material-ui/icons/PlusOne';
 import UndoIcon from '@material-ui/icons/Undo';
 import ClearIcon from '@material-ui/icons/Clear';
+import NotesIcon from '@material-ui/icons/Notes';
```

Comment:
use `ForumIcon` instead of importing new material-ui icons directly

Reply:
understood, changed